client/src/views/client-area/statistics/CustomOrderStatisticsPage.vue

```diff
@@ -0,0 +1,719 @@
+          :animated="true"
+          class="skeleton-loading-title"
+        />
+        <div class="storage-data">
+          <div class="storage-data-global">
+            <ion-skeleton-text
+              :animated="true"
+              class="skeleton-loading-number"
+            />
+            <div class="storage-data-global-info">
+              <ion-skeleton-text
+                :animated="true"
+                class="skeleton-loading-text"
+              />
+              <ion-skeleton-text
+                :animated="true"
+                class="skeleton-loading-text"
+              />
+              <ion-skeleton-text
+                :animated="true"
+                class="skeleton-loading-text"
+              />
+            </div>
+          </div>
+          <div class="storage-data-consumption">
+            <ion-skeleton-text
+              :animated="true"
+              class="skeleton-loading-text"
+            />
+            <div class="storage-data-consumption-content">
+              <div class="consumption-item">
+                <div
+                  id="firstBar"
+                  class="consumption"
+                >
+                  <div class="consumption-number">
+                    <ion-skeleton-text
+                      :animated="true"
+                      class="skeleton-loading-text"
+                    />
+                    <ion-skeleton-text
+                      :animated="true"
+                      class="skeleton-loading-text"
+                    />
+                  </div>
+                  <ion-skeleton-text
+                    :animated="true"
+                    class="skeleton-loading-text"
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </template>
+    <template v-else-if="!querying && !organizationStats && !error">
+      {{ $msTranslate('clientArea.statistics.noStats') }}
+    </template>
+    <ion-text
+      class="statistics-error body"
+      v-show="error"
+    >
+      {{ $msTranslate(error) }}
+    </ion-text>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { BmsAccessInstance, DataType, BmsOrganization, OrganizationStatsResultData, CustomOrderDetailsResultData } from '@/services/bms';
+import { onMounted, ref, computed } from 'vue';
+import { IonCard, IonText, IonIcon, IonTitle, IonSkeletonText } from '@ionic/vue';
+import { formatFileSize } from '@/common/file';
+import { person, star, pencil, arrowForward } from 'ionicons/icons';
+import { isDefaultOrganization } from '@/views/client-area/types';
+import { MsReportTheme, MsReportText } from 'megashark-lib';
+
+const props = defineProps<{
+  organization: BmsOrganization;
+}>();
+
+const organizationStats = ref<OrganizationStatsResultData | undefined>(undefined);
+const contractDetails = ref<CustomOrderDetailsResultData | undefined>(undefined);
+const useStoragePercentage = ref<number>(0);
```

Comment:
I think all those variables and functions could be replaced by computed. This way we avoid defining a variable and a function just to set that variable and we avoid having `get` functions that set values/

```
const storageSize = computed(...);
const storageWidth = computed(...);
const remainingStorageSize = computed(...);
const storagePercentage = computed(...);
...
```